lib/dynflow/executors/parallel/pool.rb

```diff
@@ -62,6 +71,12 @@ def start_termination(*args)
           try_to_terminate
         end
 
+        def execution_items(execution_plan_id = nil)
```

Comment:
Since we added more things than just the execution items. something more generic, such as `execution_status` or something like that would match more what it actually does